src/components/profile/Complete/index.js

```diff
@@ -0,0 +1,272 @@
+      dispatch(completeProfile(state));
+    }
+  }, [errors]);
+
+  const profile = (
+    <Row>
+      <Col md='6' className='p-3 background-form'>
+        <h3 className='text-center text-color'>Complete Your Profile</h3>
+        <Form onSubmit={onSubmit}>
+          {backMsg ? (
+            <Alert color='success' className='text-center'>
+              {backMsg}
+            </Alert>
+          ) : (
+            ''
+          )}
+          {getBackErrors ? (
+            <Alert color='danger' className='text-center'>
+              {getBackErrors}
+            </Alert>
+          ) : (
+            ''
+          )}
+          <Row className='mb-3'>
+            <Col md='6'>
+              <FormGroup>
+                <Label className='text-color'>GENDER</Label>
+                <Input
+                  type='text'
+                  name='gender'
+                  id='gender'
+                  onChange={onChange}
+                  className={errors.gender ? `border-error` : ''}
+                  placeholder='eg: Male'
+                />
+
+                {errors.gender ? (
+                  <Alert color='danger' className='back-error'>
+                    {errors.gender}
+                  </Alert>
+                ) : (
+                  ''
+                )}
+              </FormGroup>
+              <FormGroup>
+                <Label className='text-color'>LANGUAGE</Label>
+                <Input
+                  type='text'
+                  name='language'
+                  id='language'
+                  onChange={onChange}
+                  className={errors.gender ? `border-error` : ''}
+                  placeholder='eg: English'
+                />
+                {errors.language ? (
+                  <Alert color='danger' className='back-error'>
+                    {errors.language}
+                  </Alert>
+                ) : (
+                  ''
+                )}
+              </FormGroup>
+              <FormGroup>
+                <Label className='text-color'>COUNTRY OF RESIDENCE</Label>
+                <Input
+                  type='text'
+                  name='country'
+                  id='country'
+                  onChange={onChange}
+                  className={errors.gender ? `border-error` : ''}
+                  placeholder='eg: USA'
+                />
+                {errors.country ? (
+                  <Alert color='danger' className='back-error'>
+                    {errors.country}
+                  </Alert>
+                ) : (
+                  ''
+                )}
+              </FormGroup>
+              <FormGroup>
+                <Label className='text-color'>DEPARTMENT</Label>
+                <Input
+                  type='select'
+                  name='department'
+                  id='department'
+                  onChange={onChange}
+                  className={errors.gender ? `border-error` : ''}
+                >
```

Comment:
I think every user will need to complete a profile. It would be better if you add a note that this is reserved for the people who work at barefoot nomad to remove doubts to the users

Reply:
I don't understand.what do you mean?